armsrc/Makefile

```diff
@@ -16,7 +16,7 @@ APP_CFLAGS  = -DON_DEVICE \
 include ../common/Makefile_Enabled_Options.common
 
 ifneq (,$(findstring LCD,$(APP_CFLAGS)))
-        SRC_LCD = fonts.c LCD.c
+        SRC_LCD = #fonts.c LCD.c
```

Comment:
Huh? Will this work?

Reply:
The makefile has long included reference to SRC_LCD below when it's definition has been commented out above.  I just commented it out here as instead of above because I have no idea if those files actually work...

Reply:
You'd need to uncomment this if you want to dev the lcd.

Reply:
At first I thought the LCD.c and fonts files didn't actually exist...  but they do so I guess I didn't need to comment it out.

Reply:
the more i look the more i agree i don't like it.  un-commented it and cleaned up notes about it in the Makefile_Enabled_Options.common file.